automerge-protocol/src/lib.rs

```diff
@@ -214,6 +217,13 @@ impl ScalarValue {
                     datatype,
                 }),
             },
+            (DataType::Bytes, ScalarValue::Bytes(bytes)) => Ok(ScalarValue::Bytes(bytes.clone())),
+            (DataType::Bytes, v) => Err(error::InvalidScalarValue {
+                raw_value: self.clone(),
+                expected: "a base64 encoded string of bytes".to_string(),
```

Comment:
I think this needs updating since we aren't using b64 anymore

Reply:
Fixed